recipes-multimedia/gstreamer/gstreamer1.0-plugins-bad_%.bbappend

```diff
@@ -0,0 +1,8 @@
+# GObject introspection needs to run some commands on the native architecture,
+# and uses qemu for this. For aarch64, these commands cause qemu to crash, so
+# we disable introspection.
+EXTRA_OECONF_aarch64 += "--disable-introspection"
+
+# Wayland support is broken, so remove this until it is fixed upstream. Fixes:
```

Comment:
Is this part something that should go into like "temporary patches" if we believe it will be fixed? I'm not sure we still use the pattern of highlighting things as temporary though, @tobsan might know.

Reply:
It's likely to be fixed at some point upstream.. I have seen discussions around it on the OE mailing list, so people are working on it.

Reply:
Same goes for the introspection fixes, really.

Reply:
I'm not sure what the current process for handling these things are. The upside of using `recipes-temporary-patches` or similar is that we always know where these are at. On the other hand, they end up away from their more logical location.

Reply:
Previously, in `meta-bistro`, we would stage these bbappends in the same locations as in their original layer instead of using a temporary patches folder. The contributor and maintainers were responsible to follow up with upstream and ensure these patches get included and the temporary patches are removed. Though this caused a lot of cruft over the years.

Reply:
AFAIK we never changed the principle of "highlighting", or gathering, patches and tweaks that we see as temporary because of pending changes upstream (for example) in a common place named accordingly. The main reason was to avoid cruft. Maybe @GordanM knows about some rationale behind moving away from that, unless it was just a lack of information?

The "temporayness" of a patch was never defined, it's mostly an evaluation. I don't think that every fix should be considered temporary. If there's some explicit intent upstream to fix something, or if we have a patch that is pending merge, then the patch in the layer is temporary IMO.

I think you guys can decide what's best, I only hope we are consistent whatever the decision is. And that we document the pattern and the rationale to avoid de-facto ways of working that differs from the agreed ways of working.

Reply:
Yeah, i would actually vote for using the temporary patches directory way of doing this, just so that one can easily find everything that is supposed to be synced/patched/fixed upstream eventually.

Reply:
Alright, I will send a v2 and move this bbappend into a new recipe directory called "recipes-temporary-patches/gstreamer/" - sounds OK?